lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -55,6 +59,7 @@ public function register_routes() {
 				'args' => array(
 					'reassign' => array(),
 				),
+				'schema'          => $this->get_item_schema(),
```

Comment:
Let's not add more parameters to `register_rest_route`. Can we just add it to the existing route stuff?

```php
register_rest_route( 'wp/v2', '/comments', array(
	array(
		'methods'   => WP_REST_Server::READABLE,
		...
	),
	array(
		'methods'  => WP_REST_Server::CREATABLE,
		...
	),
	'schema' => ...
) );
```